### ISSUE TITLE
1.13 immediately crashes on Windows

### ISSUE THREAD

**Issue by bilditup1** [Xpl0itU/WiiUDownloader#25]
1.12 works fine. 
Log has: 

> (WiiUDownloader.exe:5944): Gtk-WARNING **: 10:42:05.528: Could not load a pixbuf from /org/gtk/libgtk/theme/Adwaita/assets/bullet-symbolic.svg.
> This may indicate that pixbuf loaders or the mime database could not be found.
> 
> (WiiUDownloader.exe:5944): GLib-GIO-WARNING **: 10:42:05.968: win32 session dbus binary not found
> 
> (WiiUDownloader.exe:5944): GLib-GIO-CRITICAL **: 10:42:05.968: This application can not open files.
> 
> (WiiUDownloader.exe:5944): Gtk-WARNING **: 10:42:05.997: Could not find the icon 'missing-image-ltr'. The 'hicolor' theme
> was not found either, perhaps you need to install it.
> You can get a copy from:
> 	http://icon-theme.freedesktop.org/releases
> **
> Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
> Bail out! Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
> 

So looks like some dependency was inadvertently left out of the release? (That said even 1.12 has the first two lines of errors in the above log.) Diffing the unzipped 1.12 and 1.13 folders, the only thing different between the two besides the main binary is libpsl-5.dll, however replacing the 1.13 dll with the 1.12 dll doesn't allow it to run. The link to hi-color themes given in the log is a source link. There appear to be some changes related to Windows in the commit after this one, not sure if that was an attempt to address this or what

**Comment by Xpl0itU** [comment]
From my testing, 1.13 works on Windows just fine, try reextracting or redownloading the zip file again

**Comment by bilditup1** [comment]
Hi again, 
While redownloading and reextraction did fix what seems like a corrupt `libpsl-5.dll`, I found that the error persisted after I moved program root to where I usually keep them. The issue seems to be that if the root directory's path has a directory called bin in it, in other words:
`some\path\bin\WiiUDownloader-root-directory'
or
`some\path\bin\subdir\WiiUDownloader-root-directory'
the crash happens in precisely the manner I posted above, same logs and all. This happens regardless of what the name of the program's root directory is. I guess because something (Gtk?) thinks it's on a nix system as a result of checking that path at runtime? thus it ends up trying to load icons that don't exist from a location that doesn't exist? That's my best guess, I've never used GTK. The same error also occurs with 1.12, fwiw.
